frontend/src/views/WheresMyGene/components/GeneSearchBar/components/QuickSelect/index.tsx

```diff
@@ -230,13 +243,11 @@ export default function QuickSelect<
           renderOption={renderOption}
           onPaste={handlePaste}
           InputBaseProps={{
-            onChange: (
-              event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
-            ) => {
-              setInput(event.target.value);
-            },
             placeholder,
           }}
+          inputValue={input}
+          // @ts-expect-error -- We ignore the first attribute and its a pain to type
```

Comment:
we should be able to remove this now too! Thank you!

Reply:
You'll have to explain later why the SDS changes remove the downstream typing issues 🤔

Reply:
I think since SDS also declared `onInputChange`, TypeScript eventually kept both function signatures and treated them as function overloading. But somehow TS couldn't tell which function signature we wanted to use in our code, thus the typing issue

Reply:
Ah actually, TypeScript created an intersection type of both `onInputChange` interfaces instead of an overload, so the intersection is invalid 😆 

<img width="881" alt="Screen Shot 2022-04-11 at 10 24 44 PM" src="https://user-images.githubusercontent.com/6309723/162886833-cdab8fd3-9763-46ee-988f-385320d62757.png">